src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -92,6 +101,8 @@ class Microsoft_Launcher : public PowertoyModuleIface
 
     ~Microsoft_Launcher()
     {
+        logger->info("Launcher object is destroying");
```

Comment:
Perhaps we should move these strings to the namespace at the top of this file. Same for src/runner/main.

Reply:
What are the reasons to move them up?
We are not going to reuse or localize them. Plus, if one wants to find a line that logged a message they have to do an extra step.

Reply:
I don't know, that's just how we used to do it. Maybe you're right, @enricogior?

Reply:
I would be better to always make it very clear that these are localizable strings, either with a comment or moving them in their dedicated namespace, as we do in other projects.